ipatests/pytest_ipa/integration/tasks.py

```diff
@@ -148,6 +148,29 @@ def apply_common_fixes(host):
     rpcbind_kadmin_workaround(host)
 
 
+def prepare_dse_changes(host):
```

Comment:
Any problem to keep `log_level` as an argument like previous function? 

It would be easy to spot the following and think that we can change the log level in this function. 
```python
def prepare_dse_changes(host, log_level=8192):
    ...
```